spec/jobs/characterize_job_spec.rb

```diff
@@ -12,9 +12,10 @@
 
   subject { CharacterizeJob.new(generic_file.id) }
 
-  it 'spawns a CreateDerivatives job' do
+  # Now that CreateDerivativesJob calls generic_file.create_derivatives directly
+  # this test needs to be travis exempt. 
+  it 'spawns a CreateDerivatives job', unless: $in_travis do
```

Comment:
The description of this test doesn't seem to match the test.  The test is ensuring that when a CharacterizeJob is run, the CharacterizationService is run.  I don't see any code that says, "when the CharacterizeJob is run, a CreativeDerivativesJob is run".